joinmarket-webui/jam
# fix(sweep): reload wallet info after scheduled sweep

Fixes #251.

~~This PR is based on #526 - please merge #526 first.~~
This PR is based on #529 - please merge #529 first.

Before this PR, when a scheduled sweep operation ended successfully, the balance has not been updated.
After this PR, the balance will be updated as soon as the scheduler stops (either naturally or aborted).

## How to test?
Invoke a scheduled sweep to external addresses and wait for completion.
After the scheduler finishes, the balance should be updated and show _zero_ coins left (compare this to the old behaviour which would still display the balance you started with).
You can also observe the api requests in dev tools (F12) and verify that utxos are reloaded when the operation ends.
